src/testModels.js

```diff
@@ -0,0 +1,8 @@
+import models from './models'
```

Comment:
You shouldn't commit test files (tests that you use for your own).

Reply:
Test file removed.